src/components/EditSections/EditContactInfo/EditContactInfo.test.js

```diff
@@ -65,29 +66,42 @@ const props = {
   preferredContactTypeId: '001',
   intl: {
     formatMessage : jest.fn()
-  }
+  },
 };
 
 describe('Render Edit contact Information component', () => {
   it('Must be rendered', () => {
     renderEditContactInfo(props);
     expect(screen.getByText('AddressEditList')).toBeInTheDocument();
+    expect(screen.getAllByLabelText('ui-users.contact.preferredEmailCommunication')[0]).toBeInTheDocument();
```

Comment:
1. Could you write a new test with title "should render preferred email communication field"?
2. Could you also refactor the title of this test to "should render address list"?

I know you are not the author of the original test. But could you correct what is already existing here?
Best practice is to have one test one assertion. Also, a unit test title should start with "should".